parsers/cxx/cxx_tag.c

```diff
@@ -286,6 +286,9 @@ vString * cxxTagSetProperties(unsigned int uProperties)
 		vStringCatS(pszProperties,_szProperty); \
 	} while(0)
 
+	// FIXME: The property names might be too verbose. Maybe it could be a good idea
+	//        to a switch to shorten them to 1-2 letter codes?
+
```

Comment:
I think basically using long names is not bad.
Making shortere can be done in a post processor.

Reply:
OK